addons/mail/views/mail_mail_views.xml

```diff
@@ -39,7 +39,9 @@
                         </group>
                         <notebook>
                             <page string="Body" name="body">
-                                <field name="body_content" options="{'style-inline': true}"/>
+                            <field name="body_html" widget="html"
+                                   options="{'safeHtmlEdit': true}"
```

Comment:
I'm not sure _safe_ is the right word here, is it ? It makes it look like all other usages of the html field are "unsafe", like there is a security risk or something... ^^' If I understood the issue correctly, it's only "safe" in the sense that it's not going to destroy the `head` tag, right ? Or maybe that the iframe is sandboxed too ? Maybe we could call the option `sandbox` ? The meaning of the option would be that the html itself is in a sandbox (iframe) and you can only edit it from the codeview. What do you think ?

Reply:
Let's go for `sandboxedPreview`.